src/chainparams.cpp

```diff
@@ -393,10 +408,8 @@ class CRegTestParams : public CChainParams {
         consensus.nRuleChangeActivationThreshold = 108; // 75% for testchains
         consensus.nMinerConfirmationWindow = 144; // Faster than normal for regtest (144 instead of 2016)
 
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].bit = 28;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 0;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
-        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].min_activation_height = 0; // No activation delay
+        // 0x3000_0000 = bit 28 plus versionbits signalling; 0x5000_0000 = bit 38 plus VERSIONBITS_TOP_ABANDON
+        consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY] = SetupDeployment{.start = 0, .timeout = Consensus::HereticalDeployment::NO_TIMEOUT, .activate = 0x30000000, .abandon = 0x50000000};
```

Comment:
Some math thing? It's 3 and 5 here, but 4 and 6 in deploymentinfo.h.

Reply:
It's `0x3000_000` because (I think) that matches what you'd use in core to activate testdummy on regtest, ie `VERSIONBITS_TOP_BITS) | (1 << 28)`, and the abandon version is `VERSIONBITS_TOP_ABANDON | (1 <<28)`.